ztp/extra-manifests-builder/01-kubelet-conf/90-kubelet-housekeeping-interval.conf

```
@@ -0,0 +1,4 @@
+[Service]
+ExecStartPre=/usr/local/bin/extractExecStart %n /%t/%N-execstart.env ORIG_EXECSTART
+EnvironmentFile=-/%t/%N-execstart.env
+ExecStart=bash -c "${ORIG_EXECSTART} --housekeeping-interval=30s"
```

Because we're editing ExecStart and ExecStart can only have one value, we are forced to manually clear it like this, or Systemd will complain and the service won't start:
```suggestion
ExecStart=
ExecStart=bash -c "${ORIG_EXECSTART} --housekeeping-interval=30s"
```